packages/components/src/components/hds/app-footer/status-link.ts

```diff
@@ -5,30 +5,40 @@
 
 import Component from '@glimmer/component';
 import { htmlSafe } from '@ember/template';
+import type { SafeString } from '@ember/template';
```

Comment:
I'm not quite sure how to fix this. We want to type the return for `itemStyle` as it's not technically a string, but I can't quite get the import to work here.

```
Module '"@ember/template"' declares 'SafeString' locally, but it is not exported
```

Reply:
I know it's under private, but I would point to the class
```suggestion
import { SafeString } from '@ember/template/-private/handlebars';
```

Reply:
alternatively, it can be inferred using `itemStyle?: ReturnType<typeof htmlSafe>;`

Reply:
Thanks, that did work. For some reason when I had originally tried that import it didn't 🤷